Lib/test/test_poplib.py

```diff
@@ -254,7 +258,13 @@ def assertOK(self, resp):
     def setUp(self):
         self.server = DummyPOP3Server((HOST, PORT))
         self.server.start()
-        self.client = poplib.POP3(self.server.host, self.server.port, timeout=3)
+        try:
+            self.client = poplib.POP3(self.server.host, self.server.port, timeout=3)
+        except:
```

Comment:
This should be try/finally.

Reply:
When `setUp()` succeeded, `tearDown()` do it.
So this should be `except:`, not `finally:`.